hack/app-sre/build-push.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
i don't see this script in use in https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/22093/diffs ??

Reply:
Right, I've updated the commit message accordingly:

> The other new artifact is hack/app-sre/build-push.sh. This is the script that will eventually be run by app-sre's ci-int job for direct deploy. It is a subset of hack/app_sre_build_deploy.sh: it only builds the hive operator image and pushes it to quay. **(Note that this script will not be used by appsre until all environments are migrated to direct deploy.)**